harp/__init__.py

```diff
@@ -1,3 +1,5 @@
 from harp.io import REFERENCE_EPOCH, MessageType, read
 from harp.reader import create_reader
 from harp.schema import read_schema
+
+__version__ = "0.2.0"
```

Comment:
Actually I realise I do have one issue with this.

The way versioning was done before is by using git tags instead of hard-coded strings, and I'm realising I actually really like that because you don't have to worry about updating any strings until you do the release and then uploading to pip just works consistently. Same with local builds in case there are local changes setup tools assumes a patch and pre-release suffix automatically as long as you don't hard-code things.

So I guess I would rather do this the other way around and backfill the string automatically from the GitHub release, rather than having the release version be set from a string we have to manually remember to update.

Is that possible?

Reply:
I would assume it needs to be done at the same time as the tag is added? Maybe we close this PR for now. At the time I wanted to use this to automatically generate a `Deprecated` user message but I ended up dropping it anyway.